spec/i18n_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'i18n/tasks'
+
+RSpec.describe I18n do
+  let(:i18n) { I18n::Tasks::BaseTask.new }
+  let(:missing_keys) { i18n.missing_keys }
+  let(:unused_keys) { i18n.unused_keys }
+  let(:inconsistent_interpolations) { i18n.inconsistent_interpolations }
+
+  it 'does not have missing keys' do
+    expect(missing_keys).to be_empty,
+                            "Missing #{missing_keys.leaves.count} i18n keys, run `i18n-tasks missing' to show them"
+  end
+
+  it 'does not have unused keys' do
+    expect(unused_keys).to be_empty,
+                           "#{unused_keys.leaves.count} unused i18n keys, run `i18n-tasks unused' to show them"
+  end
+
+  it 'files are normalized' do
+    non_normalized = i18n.non_normalized_paths
+    error_message = "The following files need to be normalized:\n" \
+                    "#{non_normalized.map { |path| "  #{path}" }.join("\n")}\n" \
+                    "Please run `i18n-tasks normalize' to fix"
+    expect(non_normalized).to be_empty, error_message
+  end
+
+  it 'does not have inconsistent interpolations' do
+    error_message = "#{inconsistent_interpolations.leaves.count} i18n keys have inconsistent interpolations.\n" \
+                    "Run `i18n-tasks check-consistent-interpolations' to show them"
+    expect(inconsistent_interpolations).to be_empty, error_message
+  end
+end
```

Comment:
Given this is not testing behavior from our production code and instead it's just about running some tasks from a vendor gem, what do you think if we have these checks in our travis CI as a new step instead of this spec?

```yml
     - run:
          name: I18n Health
          command: bundle exec i18n-tasks health
```

Reply:
I completely agree on this, but would love to hear at least one more opinion on this. Then I can change this in next pr 😉

Reply:
@jpascual1994 @santib Any thoughts?

Reply:
I totally agree 😄

Reply:
Agree with @juanmanuelramallo on this! We should add the health command to the CI checks and avoid this test.

Reply:
Please change that on this PR :)

Reply:
@jdrosales17 @Juanma1991 done
Next time be dyslexia inclusive and send a copy & paste 😅